_pages/about-us/teams/tech-portfolio.md

```diff
@@ -3,30 +3,39 @@ title: TTS Technology Portfolio
+- Serve as the primary liaison between [GSA IT](https://www.gsa.gov/about-us/organization/gsa-it) and TTS, ensuring high levels of service, finding compromise, and generally keeping a good relationship between the teams
+
+We also do things like:
+
+- **Policy:** Represent TTS on the [FAS Systems Governance Committee (FSGC)](https://drive.google.com/file/d/18zOMBK5bhmqt-Y3yGTKQ_QiiC_x9i5va/view?usp=sharing), i.e. vote on what technology projects get money from the [Acquisition Services Fund (ASF)](https://www.gsa.gov/reference/reports/budget-performance/annual-reports/agency-financial-report-2012/managements-discussion-and-analysis/financial-statements-analysis-and-summary/financial-results-by-major-fund-acquisition-services-fund)
+- **Compliance:** Assist in [authorization](https://atos.open-control.org/steps/#step-5-authorize-information-system) of TTS systems to get their ATOs, improving that process along the way
+- **Requirements:** In coordination with GSA IT, manage and support procurements, approvals, and licenses (with Acquisition and Finance teams) for [hardware]({{site.baseurl}}/equipment/) and [software]({{site.baseurl}}/software/) that chapters need to accomplish their mission
+- **Assets:** Manage major [TTS infrastructure](https://before-you-ship.18f.gov/infrastructure/) and programs such as [Bug Bounty](https://hackerone.com/tts), [AWS](https://before-you-ship.18f.gov/infrastructure/aws/), New Relic, Twilio, etc.
+- **Relationships:** Advise TTS leadership on technology-related matters
+- **Customer Service:** Generally support TTS on any technology-related matters
+- **Administration:** Perform operational activities like access, account and license management. 
+
+Things not within our scope:
+
+- Purchasing office equipment for individuals (rather than the project the individual supports)
 
 ## Important links
 
 - [The Tech Portfolio Kanban board](https://github.com/orgs/18F/projects/11?fullscreen=true) - what we're working on
 - [Before You Ship](https://before-you-ship.18f.gov/) - TTS's internal documentation about security, compliance, and infrastructure
-- [Software-as-a-service (SaaS) in use at TTS](https://docs.google.com/spreadsheets/d/12pfcEIEXaJTjIKex-3wnI89erIvgKf9B_XpGkDl6qsM/edit#gid=0)
+- Maintaining the authoritative source documentation for components used across TTS  [Software-as-a-service (SaaS) ](https://docs.google.com/spreadsheets/d/12pfcEIEXaJTjIKex-3wnI89erIvgKf9B_XpGkDl6qsM/edit#gid=0), [Systems](https://docs.google.com/spreadsheets/u/1/d/1LGn9kZCphzf14V5HpBfV3aYjflu-k9GtHS5LmFvDywM/edit?usp=drive_web&ouid=114492559070606542558), [Accounts](https://docs.google.com/spreadsheets/d/1DedSCiU9AsCAAVvAFZT0_Ii7AFIKlI-JNifzlpHNbDg/edit#gid=0) and  [Contracts](https://docs.google.com/spreadsheets/u/1/d/1S9cUnhRwfdsxcgT_h8ynHBbOQBxHgWlN-pzQ3yHw5_E/edit?usp=drive_web&ouid=114492559070606542558) 
```

Comment:
Nitpick: the list of contracts is very relevant to us, but I don't know that we are / should be the ones to maintain it.

Reply:
I assumed that was within our scope based on being portfolio managers & for the purpose of what came up on the call w/ @JJediny in the retro today about needing to have an awareness of what is going on across the board. But I can also see the case that a different office like Acq or Finance would maintain it.

Reply:
https://github.com/18F/tts-tech-portfolio/issues/155

Reply:
Some of the contracts are in-scope for us, for sure.

Reply:
Ahh gotcha!